lib/shape/CollideCircle.js

```diff
@@ -70,4 +70,4 @@ function CollideCircles(manifold, circleA, xfA, circleB, xfB) {
   manifold.points[0].id.cf.typeB = Manifold.e_vertex;
 }
 
-exports.CollideCircles = CollideCircles;
+export { CollideCircles };
```

Comment:
Is there a reason why the collide shapes are not the default export of the module?

Reply:
These first few comments are of course purely subjective, so feel free to just resolve them 🙂